webrender/src/render_backend.rs

```diff
@@ -163,6 +166,9 @@ impl RenderBackend {
         blob_image_renderer: Option<Box<BlobImageRenderer>>,
         enable_render_on_scroll: bool,
     ) -> RenderBackend {
+        // The namespace_id should start from 1.
```

Comment:
I can't set the value to 1 directly, so I just increase the value in RenderBackend::new().

error: `std::sync::atomic::AtomicUsize::new` is not yet stable as a const fn
   --> src\render_backend.rs:129:41
    |
129 | static NEXT_NAMESPACE_ID: AtomicUsize = AtomicUsize::new(1);
    |                                         ^^^^^^^^^^^^^^^^^^^
    |
    = help: in Nightly builds, add `#![feature(const_atomic_usize_new)]` to the crate attributes to enable